installers/foxpass_setup_beanstalk.config

```diff
@@ -0,0 +1,23 @@
+option_settings:
+- namespace: aws:elasticbeanstalk:application:environment
+- option_name: FP_DNNAME
+  value: [cn=company,cn=com]
+- option_name: FP_BINDNAME
+  value: [linux|users|ldap]
+- option_name: FP_BINDPW
+  value: [vault output]
+- option_name: FP_APIKEY
+  value: [vault output]
+
+files:
+  "/tmp/foxpass_setup.py" :
+    mode: "000755"
+    owner: root
+    group: root
+    source: https://raw.githubusercontent.com/foxpass/foxpass-setup/master/linux/<DISTRO>/<VERSION>/foxpass_setup.py
+
+commands:
+  foxpass_setup:
+    command: "foxpass_setup.py --base-dn $FP_DNNAME --bind-user $FP_BINDNAME --bind-pw $FP_BINDPW --api-key $FP_APIKEY"
+    cwd: /tmp
+    test: "/usr/bin/getent group | grep foxpass-sudo"
```

Comment:
we don't make this group automatically, so this could fail even though everything is installed correctly.

Reply:
Want me to put `foxpass-sudo` in `[]` or `<>` like the other things that may need to be changed?

Reply:
Whatever you think is best... just wanted to flag it.